tests/operator/operator.go

```diff
@@ -3446,3 +3341,104 @@ func deprecatedBeforeAll(fn func()) {
 func usesSha(image string) bool {
 	return strings.Contains(image, "@sha256:")
 }
+
+func startAllVMIs(virtClient kubecli.KubevirtClient, vmis []*v1.VirtualMachineInstance) []*v1.VirtualMachineInstance {
+	newVMIs := make([]*v1.VirtualMachineInstance, len(vmis))
+	for i, vmi := range vmis {
+		foundVMI, err := virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Create(context.Background(), vmi, metav1.CreateOptions{})
+		Expect(err).ToNot(HaveOccurred(), "Create VMI successfully")
+		newVMIs[i] = libwait.WaitForSuccessfulVMIStart(foundVMI)
```

Comment:
but you still wait for each of them. would you add another `for` in the end, instead?

Reply:
missed that. done